### PR TITLE
add scalafmt and apply it

### PR DESCRIPTION
**Pull request by thinkerou**

### CODE REVIEW

File: src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala [hunk 1]
```diff
@@ -61,35 +67,36 @@ class GrpcAction[ReqT <: GenM, ResT <: GenM](grpcRequestAttributes: GrpcRequestA
         )
       } match {
         case scala.util.Success(response) => Future.successful(response)
-        case scala.util.Failure(e) => Future.failed(e)
+        case scala.util.Failure(e)        => Future.failed(e)
```
Comment (macchiatow) [inline src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala:70]: why this?

File: src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala [hunk 2]
```diff
@@ -21,23 +21,27 @@ import scala.language.postfixOps
...
-  def validateRequest[T](session: Session)(implicit reqT: ClassTag[T]): Validation[T] = {
+  def validateRequest[T](session: Session)(
+      implicit reqT: ClassTag[T]): Validation[T] = {
     grpcRequestAttributes.request(session) flatMap {
       case req: T =>
         io.gatling.commons.validation.Success(req)
       case req =>
-        val err = s"Feeder type mismatch: required $reqT, but found ${req.getClass}"
+        val err =
+          s"Feeder type mismatch: required $reqT, but found ${req.getClass}"
```
Comment (macchiatow) [inline src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala:44]: this new line doesn't look natural, does it?

**Comment by macchiatow** [comment]
I use scalastyle from, I believe, google imported into Intellij, it looks good to me. overall I don't like to many "new line" if there no particular reason.

File: src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala (hunk 1 above)
Comment (thinkerou) [reply src/main/scala/me/macchiatow/gatling/grpc/action/GrpcAction.scala:70]: thanks, the line is scalafmt result.

**Comment by thinkerou** [comment]
OK, thanks @macchiatow , I now use scalafmt, I learn scalastyle later on.

**Comment by thinkerou** [comment]
close it, if need reopen.

**Pull request closed without merging**